src/Compilers/CSharp/Test/Syntax/Parsing/AnonymousFunctionParsingTests.cs

```diff
@@ -2103,7 +2091,7 @@ void verify()
                             }
                         }
                     }
-                    M(SyntaxKind.SemicolonToken);
+                    N(SyntaxKind.SemicolonToken);
```

Comment:
Lots less missing tokens as error recovery now does a better job not preemptively bailing out.